config/jobs/kubernetes/sig-scalability/sig-scalability-golang.yaml

```diff
@@ -40,7 +40,6 @@ periodics:
       - runner.sh
       args:
       - make
-      - --
```

Comment:
is the --directory a flag of `make`

Reply:
yes, it's the long name for `-C`

Reply:
https://www.gnu.org/software/make/manual/html_node/Options-Summary.html

Reply:
execute.py only had a single flag, `--env`, which was not used by any of these jobs